ios 15 push (#2571)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/push/ios/notification_options.md

  {% endtab %}
{% endtabs %}

### Interruption Level (iOS 15+) {#interruption-level}

With iOS 15's new Focus Mode, users are more in control over when app notifications can "interrupt" them with a sound or vibration.

Apps can now specify what level of interruption a notification should include, based on its urgency.

![Notification Settings]({% image_buster /assets/img/ios/ios15-notification-settings.png %}){: style="float:right;max-width:25%;margin-left:15px;border:0"}

> Keep in mind that users are ultimately the ones in control of their focus, and even if a Time Sensitive notification is delivered, they can specify which apps are not allowed breaking through their focus.

Reviewer: ```suggestion
Keep in mind that users are ultimately the ones in control of their focus, and even if a Time Sensitive notification is delivered, they can specify which apps are not allowed to break through their focus.
```

|[Critical](https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel/critical)|Will make a sound, vibrate, and turn on the screen even if the phone's Do Not Disturb switch is enabled. This [requires explicit approval by Apple](https://developer.apple.com/contact/request/notifications-critical-alerts-entitlement/).|Emergencies such as severe weather or safety alerts|Yes|

To change the Interruption Level for an iOS Push Notification, choose the desired level from the Settings tab within your message:

![Interruption Level Option][28]

### Relevance Score (iOS 15+) {#relevance-score}

![Notification Summary]({% image_buster /assets/img/ios/ios15-notification-summary.png %}){: style="float:right;max-width:25%;margin-left:15px;border:0"}

iOS 15 also introduces a new way for users to (optionally) schedule a digest grouping of multiple notifications at designated times throughout the day.

This is done to prevent constant interruptions throughout the day for notifications which don’t need immediate attention.

> Apps can specify which push notifications are most relevant, by setting a __Relevance Score__. Apple will use this score to determine which notifications should be showcased in the scheduled Notification Summary while others are made available when users click into the summary. 

Reviewer: ```suggestion
Apps can specify which push notifications are most relevant by setting a __Relevance Score__. Apple will use this score to determine which notifications should be showcased in the scheduled Notification Summary while others are made available when users click into the summary. 
```

All notifications will still be accessible in the user's notification center.

> To set an iOS Notification's Relevance Score, enter a value between `0.0` and `1.0` within the Settings tab. For example, the most important message should be set with `1.0`, whereas a medium-importance message can be sent with `0.5`.

Reviewer: ```suggestion
To set an iOS Notification's Relevance Score, enter a value between `0.0` and `1.0` within the Settings tab. For example, the most important message should be sent with `1.0`, whereas a medium-importance message can be sent with `0.5`.
```
